assets/css/styleguide.scss

```diff
@@ -128,7 +130,16 @@ $site-top:           124px;
   display: inline;
   float: left;
   margin-top: -4px;
-  padding-left: 20px;
+  padding-top: 15px;
```

Comment:
@maya Fixed! Sorry should not do a PR so late! :P Let me know if I missed anything else!